src/patch.js

```diff
@@ -22,6 +22,20 @@ var walker = require('./walker'),
     getWalker = walker.getWalker,
     setWalker = walker.setWalker;
 
+/**
+ * @type {boolean}
+ * @const
+ */
+var IS_PRODUCTION = process.env.NODE_ENV === 'production';
+
+if (!IS_PRODUCTION) {
+  var assertNoUnclosedTags = function() {
+    if (getWalker().getCurrentParent()) {
+      throw new Error('One or more tags were not closed');
```

Comment:
I could print the parent path to the patched node, which should be the same as the private static held by the walker.